services-directions/src/test/java/com/mapbox/api/directions/v5/models/BannerInstructionsTest.java

```diff
@@ -29,8 +29,8 @@ public void sanity() throws Exception {
   public void testSerializable() throws Exception {
     BannerInstructions bannerInstructions = BannerInstructions.builder()
       .distanceAlongGeometry(100d)
-      .primary(BannerText.builder().build())
-      .secondary(BannerText.builder().build())
+      .primary(BannerText.builder().text("Banner primary sample text").build())
+      .secondary(BannerText.builder().text("Banner secondary sample text").build())
```

Comment:
If `secondary` is `@Nullable`, this is not needed right? I mean, passing a `null` value (as before) should work too.

Reply:
Thanks for checking in @Guardiola31337 . Looks like `null` is fine as a parameter.

<img width="1052" alt="Screen Shot 2019-04-10 at 2 01 08 PM" src="https://user-images.githubusercontent.com/4394910/55913275-1d3a6180-5b99-11e9-824d-060356e92561.png">

Reply:
Yeah and we'd be implicitly "testing" that `secondary` could be `@Nullable`, thanks for confirming @langsmith